flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/FlinkPlannerImpl.scala

```diff
@@ -60,6 +60,7 @@ class FlinkPlannerImpl(
   var root: RelRoot = _
 
   private def ready() {
+    planner.setExecutor(config.getExecutor)
```

Comment:
I would not set the planner here as this class is only used by the SQL API. `FlinkRelBuilder.create()` is a better place.